mpl_typst/backend.py

```diff
@@ -387,22 +445,32 @@ def print_png(self, filename, **kwargs):
     def print_svg(self, filename, **kwargs):
         return self._print_as('svg', filename, **kwargs)
 
-    def print_typ(self, filename, *, metadata=None, **kwargs):
+    def print_typ(self, filename, *, bbox_inches_restore=None, metadata=None,
+                  **kwargs):
         # TODO(@daskol): Matplotlib shows quite unexpectedbehaviour. It renders
         # the same figure multiple types with randering to temporary buffer
         # (BytesIO) rather than directly to file. So, it would be great to
         # rewrite this function and neighnoring one to make it file-agnostic.
+        def _render_to_stream(buf):
+            width, height = self.figure.get_size_inches()
+            dpi = self.figure.dpi
+            self.figure.dpi = 72
```

Comment:
Hey, it would be great to have this feature implemented! It seems to me the CI is not passing just because the test figure was saved with a different dpi from the current one (I imagine it was saved with `dpi = 100` which is matplotlib's default). Changing this to `self.figure.dpi = 100` or redoing the test figure solves the CI.
```suggestion
            self.figure.dpi = 100
```

Reply:
I play with this PR a little bit. Proper definition of `get_image_magnification()` solves the issue partially but I've observed that the resulting image is misplaced by a point or a half.

Reply:
I have observed the same